packages/office-ui-fabric-react/src/components/Label/Label.scss

```diff
@@ -15,7 +15,7 @@
   color: $ms-color-neutralPrimary;
   box-sizing: border-box;
   display: block;
-  padding: 5px 0;
+  padding: 8px 0;
```

Comment:
@dzearing  Maybe we can leave the top padding at 5px and just change the bottom to 8px? Or maybe just close this with a won't fix?